dev.deedles.Trayscale.yml

```diff
@@ -0,0 +1,49 @@
+app-id: dev.deedles.Trayscale
+
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: trayscale
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --filesystem=/run/tailscale:ro
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Flatpak
+
+build-options:
+  build-args:
+    - --filesystem=home # To access the Go build cache.
```

Comment:
which cache?

Reply:
The Go toolchain caches previously compiled packages. The default location is `$XDG_CACHE_HOME/go-build`. It was just to make repeat builds on the local system faster. If it's an issue, I can remove it.

Reply:
Is there any way to pass them to `flatpak-builder` on the command line so that I can replicate the effect locally?

Reply:
On a related note, is there a way to specify the version for entries in `sdk-extensions`? I couldn't find anything in the documentation. The current one seems to be defaulting to `22.08` which is a version of Go behind.

Reply:
22.08 is the latest SDK, and matches GNOME 43. There is no way to specify a specific go version. 

The SDK should be updated soon. (it should have)